### PR TITLE
Add labelArc property to allow for some label curvature

### PR DESCRIPTION
**Pull request by ohmec**
Adding a feature to "prettify" some labels where it makes sense. This can be useful for borderless properties to give a better sense of "span" without the borders. But can also be useful in other instances as shown in this Massachusetts Bay example.

The property `labelArc` can be added to arc the text in either a downward (positive) or upward (negative) direction. The value of the property indicates the direction and radius of the arc, with larger positive (or smaller negative) values having less curvature.

The documentation for the extended GeoJSON specification has been updated with this definition, but an example before and after is shown below.

Signed-off-by: Scott Johnson <sjcupertino@gmail.com>

### CODE REVIEW

**Comment by ohmec** [comment]
@warrenjp when you have time it would be good for you to review this as well as the Extended GeoJSON Specification additions for `labelArc` to make sure they make sense.

File: ohmec.js [hunk 1]
```diff
@@ -208,7 +208,22 @@ function onEachFeature(feature, layer) {
...
+    // it is a circle with radius 'arc' that has a tangent at (50,h/2), either with the circle below
+    // and the text on the top (if arc > 0) or the circle above with the text on the bottom (arc < 0).
+    if("labelArc" in feature.properties) {
+      let arcval = feature.properties.labelArc;
+      let my   = heightd2 + 2*arcval;
+      let ar   = (arcval >= 0) ? arcval : -arcval;
+      let pos  = (arcval >= 0) ?  1 :   0;
+      let ar2n = arcval*2;
+      let ar2p = arcval*-2;
+      inner += '<path id="arcpath' + feature.id + '" stroke="none" fill="none" d="m 50,' + my;
```
Comment (ohmec) [inline ohmec.js:222]: Found a bug in this. The SVG namespace is evidently global, so needed to uniquify the path ID with the feature ID.

Comment (ohmec) [reply ohmec.js:222]: For the record, it only appears to be an issue when two SVGs are active at a time. But still this is the right fix.

**Comment by ohmec** [comment]
> I'd like to pull down the new code into my current working branch and use these new capabilities. This will also enable me to assess the Extended GeoJSON Specification. @ohmec How do I update my current branch? Is this a rebase?

This from the Github Primer should help. If not, please assist by clarifying that document. 😄 

---

**Visualizing another user’s Pull Request**

It is common while reviewing another’s code to want to visualize it on your side, by creating a version of it locally and seeing how it renders in the browser, for instance. This can be done easily by referring to the Pull Request. All that is needed is the number of the PR, which in this example is `123`. It should be noted that this command can be executed from any branch, but if that branch has outstanding work on it, you should execute `git add -u` and `git commit` in advance of this fetch.

```sh
unix 36> git fetch upstream pull/123/head:localpr_123
unix 37> git branch
  main
  dev_labels
  dev_guat1
* localpr_123
```

This example has fetched another user’s PR #123, and put it into a branch (arbitrarily) called `localpr_123`. From there you can render the contents of the PR and/or make modifications locally.

**Review by warrenjp — APPROVED** [review]
Tested successfully. Certainly possible that issues will crop up, but expected behavior was observed on first effort.

**Comment by warrenjp** [comment]
Screen Shot 2021-10-12 at 10 45 44 AM

**Pull request merged**